packages/web/src/lib/core/StitchAppClient.ts

```diff
@@ -12,7 +12,7 @@ interface StitchAppClient {
 
   getServiceClient<T>(provider: ServiceClientProvider<T>): T;
 
-  callFunction(name: string, ...args: any[]): Promise<any>;
+  callFunction(name: string, args: any[]): Promise<any>;
```

Comment:
[note] let's make a note of this for when we play around with usability next week. Since this is not internal, I think this could go either way here for variadic vs. non-variadic.

Reply:
To be honest, I ran into the exact issue we were talking about– doubling up the use of an array. I'm not sure variadic works as well in a language as loose as TS.